spec/presenters/person_presenter_spec.rb

```diff
@@ -142,4 +143,83 @@
       expect(presenter.hovercard[:contact]).to have_key(:aspect_memberships)
     end
   end
+
+  describe "#profile_hash_as_api_json" do
+    let(:current_user) { FactoryGirl.create(:user,
```

Comment:
Layout/MultilineBlockLayout: Block body expression is on the same line as the block start.